components/ca-tel.js

```diff
@@ -0,0 +1,175 @@
+define([
+    './helpers/create-element.js',
+    './helpers/fire-event.js',
+    './ca-select.js',
```

Comment:
Agree on the black box.  If we changed from ca-select to ca-blah other internal impl, then all code using ca-tel should not need to be updated.